### DISCUSSION TITLE
Error al autenticarme para enviar las tareas

### DISCUSSION THREAD

**Discussion by gitogel** [rramosp/ai4eng.v1#247] [00 - General]
Buenas Profesor:
No puedo autenticarme para enviar las soluciones a los talleres. Me sale este error: 
logging in as [gelzaalba@gmail.com](mailto:gelzaalba@gmail.com) ... please wait

login failed!!!

you are registered in the following course sessions

   course                                   session             
   -----------------                        ------------------- 
   ai4eng.v1.udea.r2                        2023.1              
   ai4eng.v1                                open
run this cell specifying the course and session you want to login. For instance:

        session.LoginSequence(endpoint=init.endpoint, course_id="ai4eng.v1.udea.r2", session_id="2023.1",....)

Creo que por error me inscribí dos veces en el curso y tal vez eso esté dando problemas. ¿Podría ayudarme?

**Comment by gitogel** [comment]
Cambié la sesión y el curso, me dejó autenticarme, pero no puedo subir los resultados, me da este error:
student.submit_task(globals(), task_id="task_02");
NameError                                 Traceback (most recent call last)
[<ipython-input-17-fde445fafb4e>](https://localhost:8080/#) in <module>
----> 1 student.submit_task(globals(), task_id="task_02");

NameError: name 'student' is not defined

**Comment by gitogel** [comment]
Hola, alguna respuesta?

**Comment by imEag** [comment]
Hola, vi en otro post que cambiando de usuario se solucionó, sin embargo me gustaría saber si eres estudiante actual del curso de la UdeA? Si es así ¿creaste la cuenta también con el correo universitario? y si, si ¿tienes problemas con esa cuenta? 

¿o ya quedó todo solucionado como te gustaría?
